### PR TITLE
new page: advanced account management

### PR DESCRIPTION
**Pull request by dandanlen**
Contains info about address binding and balance restrictions.

### CODE REVIEW

File: pages/perseverance/funding/advanced-account-management.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+description: These are some features that advanced operators should be aware of.
+
+---
+
+import { Callout } from "@/components";
+
+## Redeem Address Binding
+
+Any account may be explicitly *bound* to a single redemption address. Henceforth, any redemption request from that account can *only redeem to this exact address*.
```
Comment (fernandezdavid) [inline pages/perseverance/funding/advanced-account-management.mdx:13]: This should be the description of the page I'd say. It's fine to repeat 👌

File: pages/perseverance/funding/advanced-account-management.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+> Note: At the time of writing, executor binding via the CLI is not yet supported on Perseverance network. This feature is planned for release as of version 0.9.4.
+
+Similar to the above, redemptions can be restricted such that only a single specific Ethereum address can be the *executor* of a redemption.
+
+In order to bind an executor address, proceed as above using the `chainflip-cli`.
+
+Query for any currently bound address like so:
+
+```bash
```
Comment (fernandezdavid) [inline pages/perseverance/funding/advanced-account-management.mdx:51]: you can do:
```bash copy

To include the copy icon on hover (same for the ones below)

File: pages/perseverance/funding/advanced-account-management.mdx (hunk 1 above)
Comment (dandanlen) [reply pages/perseverance/funding/advanced-account-management.mdx:17]: Ok - I though that's what the Title/Description stuff was above. Where does that end up?

Comment (fernandezdavid) [reply pages/perseverance/funding/advanced-account-management.mdx:17]: The `description` attribute at the top of the page ends up as the html tag `<meta name="description" content="X">`, where 'X' should be the best summary of the content of the page and its used by search engines. 

In this case, I thought your first line under 'Redeem Address Binding' h2 best describes the page's content and should go as `description` instead.

**Review by fernandezdavid — APPROVED** [review]

File: pages/perseverance/funding/advanced-account-management.mdx (hunk 1 above)
Comment (dandanlen) [reply pages/perseverance/funding/advanced-account-management.mdx:17]: It covers multiple topics, so I figured it put it all under one heading.. hope that's ok. Happy to edit later to something more specific when we add some more stuff.

**Pull request merged**